### PR TITLE
Add support for multi platform images

### PR DESCRIPTION
**Pull request by MilkyWare**
Hi,

This is kind of in relation to #345 

Been wanting to use this image to have a web UI to manage my redis install on a raspberry pi, however on docker hub it states the latest tag only support Linux/amd64 despite the fact that the alpine image this is based on supports more platforms.

I don't know what you guys use for publishing to docker hub currently, but I've had a look at GitHub actions and built a docker buildx action which pushes x86, x64 and arm images to docker hub. For an example please feel free to look at [my docker hub](https://hub.docker.com/repository/docker/milkyjoe93/redis-commander/tags?page=1)

This action will automatically build on each commit to master but also support manual triggers.

Hope this is useful
Cameron

### CODE REVIEW

**Comment by sseide** [comment]
Hi - this PR is really good to have automated builds for all architectures. But setting up secrets needs to be done by @joeferner [1] Before this is done its no use to merge it...


[1] https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets

**Comment by isaacrlevin** [comment]
This is awesome, would love to get Commander on my pi server. Any ETA on publishing to Docker Hub @joeferner

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          version: latest
+          
+      - name: Docker login
+        uses: docker/login-action@v1.4.1
+        with: 
+          username: "${{ secrets.DOCKER_USERNAME }}"
+          password: "${{ secrets.DOCKER_PASSWORD }}"
+        
+      - name: Docker build and push
+        uses: docker/build-push-action@v2
```
Comment (thomastvedt) [inline .github/workflows/docker-publish.yml:43]: @v2.3.0 ?

**Comment by thomastvedt** [comment]
And we could add proper versioning too in the same go? See here https://github.com/joeferner/redis-commander/pull/441/files

add:
```
      - name: Get latest tag
        id: tag
        run: |
          git fetch --prune --unshallow
          echo '::set-output name=version::'$(git describe --tags $(git rev-list --tags --max-count=1))
```


change build and push step to push versioned tag + latest:

```

      - name: Docker build and push
        uses: docker/build-push-action@v2.3.0
        with:
          context: .
          platforms: linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/386,linux/amd64
          push: true
          tags: |
            ${{ secrets.DOCKER_REPO }}/redis-commander:latest,
            ${{ secrets.DOCKER_REPO }}/redis-commander:${{steps.tag.outputs.version}}
```

**Comment by MilkyWare** [comment]
Hi @thomastvedt. With your comments in mind, I've added automatic git tagging and creating a secondary docker hub tagging using this. Once again, I'm not sure how you guys use versioning, but can always be tweaked

**Comment by thomastvedt** [comment]
> Hi @thomastvedt. With your comments in mind, I've added automatic git tagging and creating a secondary docker hub tagging using this. Once again, I'm not sure how you guys use versioning, but can always be tweaked

Great thanks @MilkyWare , I'll close my other PR and hopefully @joeferner will incorporate changes in this one 👍🏻

**Comment by novikov1337danil** [comment]
@joeferner hope for the adoption of this PR

**Comment by carromeu** [comment]
Any chance to adopt this request and launch in next release? Has other way to run this image in ARM64?

**Pull request closed without merging**